specification/resources/registry/multiregistry_create.yml

```
@@ -0,0 +1,51 @@
+operationId: multiregistry_create
+
+summary: Create Container Registry By Name (Beta)
```

```suggestion
summary: [Beta] Create Container Registry By Name
```

Let's put the beta tag up front for clarity.

---

Noted. I had to put it inside quotes or else the operation was not visible in the rendered spec.

---

Hey @andrewsomething, closing this as it has been updated.